[Bug]: Failed to compile the procps package

### Problem description

An error occurred when compiling the procps package
Because I want to use bash in another application
```
/home/builder/.termux-build/procps/src/pidof.c:213:14: warning: address of array 'task.cmd' will always evaluate to 'true' [-Wpointer-bool-conversion]
                                if (task.cmd &&
                                    ~~~~~^~~ ~~
/home/builder/.termux-build/procps/src/w.c:229:43: error: use of undeclared identifier '__UT_HOSTSIZE'
                        print_display_or_interface(u->ut_host, UT_HOSTSIZE, fromlen - len);
                                                               ^
/home/builder/.termux-build/procps/src/w.c:76:22: note: expanded from macro 'UT_HOSTSIZE'
#       define UT_HOSTSIZE __UT_HOSTSIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:231:27: error: use of undeclared identifier '__UT_HOSTSIZE'
                        print_host(u->ut_host, UT_HOSTSIZE, fromlen);
                                               ^
/home/builder/.termux-build/procps/src/w.c:76:22: note: expanded from macro 'UT_HOSTSIZE'
#       define UT_HOSTSIZE __UT_HOSTSIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:234:26: error: use of undeclared identifier '__UT_HOSTSIZE'
                print_host(u->ut_host, UT_HOSTSIZE, fromlen);
                                       ^
/home/builder/.termux-build/procps/src/w.c:76:22: note: expanded from macro 'UT_HOSTSIZE'/home/builder/.termux-build/procps/src/watch.c:432:15: warning: result of comparison of constant -1 with expression of type 'char' is always false [-Wtautological-constant-out-of-range-compare]
                if (i[byte] == EOF) {
                    ~~~~~~~ ^  ~~~

#       define UT_HOSTSIZE __UT_HOSTSIZE
                           ^
  CC       lib/watch-strutils.o
/home/builder/.termux-build/procps/src/w.c:350:12: error: use of undeclared identifier '__UT_NAMESIZE'
                char buf[UT_NAMESIZE + 1];
                         ^
/home/builder/.termux-build/procps/src/w.c:78:22: note: expanded from macro 'UT_NAMESIZE'
#       define UT_NAMESIZE __UT_NAMESIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:353:28: error: use of undeclared identifier '__UT_NAMESIZE'
                strncpy(buf, u->ut_user, UT_NAMESIZE);
                                         ^
/home/builder/.termux-build/procps/src/w.c:78:22: note: expanded from macro 'UT_NAMESIZE'
#       define UT_NAMESIZE __UT_NAMESIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:354:7: error: use of undeclared identifier '__UT_NAMESIZE'
                buf[UT_NAMESIZE] = '\0';
                    ^
/home/builder/.termux-build/procps/src/w.c:78:22: note: expanded from macro 'UT_NAMESIZE'
#       define UT_NAMESIZE __UT_NAMESIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:395:13: error: use of undeclared identifier '__UT_NAMESIZE'
        char uname[UT_NAMESIZE + 1] = "", tty[5 + UT_LINESIZE + 1] = "/dev/";
                   ^
/home/builder/.termux-build/procps/src/w.c:78:22: note: expanded from macro 'UT_NAMESIZE'
#       define UT_NAMESIZE __UT_NAMESIZE
                           ^
  CC       lib/watch-fileutils.o
/home/builder/.termux-build/procps/src/w.c:395:44: error: use of undeclared identifier '__UT_LINESIZE'
        char uname[UT_NAMESIZE + 1] = "", tty[5 + UT_LINESIZE + 1] = "/dev/";
                                                  ^
/home/builder/.termux-build/procps/src/w.c:77:22: note: expanded from macro 'UT_LINESIZE'
#       define UT_LINESIZE __UT_LINESIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:395:60: error: expected ';' at end of declaration
        char uname[UT_NAMESIZE + 1] = "", tty[5 + UT_LINESIZE + 1] = "/dev/";
                                                                  ^
                                                                  ;
/home/builder/.termux-build/procps/src/w.c:398:18: error: use of undeclared identifier '__UT_LINESIZE'
        for (i = 0; i < UT_LINESIZE; i++)
                        ^
/home/builder/.termux-build/procps/src/w.c:77:22: note: expanded from macro 'UT_LINESIZE'
#       define UT_LINESIZE __UT_LINESIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:401:4: error: use of undeclared identifier 'tty'
                        tty[i + 5] = u->ut_line[i];
                        ^
/home/builder/.termux-build/procps/src/w.c:403:4: error: use of undeclared identifier 'tty'
                        tty[i + 5] = '\0';
                        ^
/home/builder/.termux-build/procps/src/w.c:405:20: error: use of undeclared identifier 'tty'
        best = getproc(u, tty + 5, &jcpu, &ut_pid_found);
                          ^
/home/builder/.termux-build/procps/src/w.c:417:29: error: use of undeclared identifier '__UT_NAMESIZE'
        strncpy(uname, u->ut_user, UT_NAMESIZE);
                                   ^
/home/builder/.termux-build/procps/src/w.c:78:22: note: expanded from macro 'UT_NAMESIZE'
#       define UT_NAMESIZE __UT_NAMESIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:432:30: error: use of undeclared identifier 'tty'
                        print_time_ival7(idletime(tty), 0, stdout);
                                                  ^
/home/builder/.termux-build/procps/src/w.c:451:30: error: use of undeclared identifier 'tty'
                        print_time_ival7(idletime(tty), 0, stdout);
                                                  ^
/home/builder/.termux-build/procps/src/w.c:569:21: error: use of undeclared identifier '__UT_NAMESIZE'
                int ut_namesize = UT_NAMESIZE;
                                  ^
/home/builder/.termux-build/procps/src/w.c:78:22: note: expanded from macro 'UT_NAMESIZE'
#       define UT_NAMESIZE __UT_NAMESIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:581:22: error: use of undeclared identifier '__UT_HOSTSIZE'
                if (fromlen < 8 || UT_HOSTSIZE < fromlen) {
                                   ^
/home/builder/.termux-build/procps/src/w.c:76:22: note: expanded from macro 'UT_HOSTSIZE'
#       define UT_HOSTSIZE __UT_HOSTSIZE
                           ^
/home/builder/.termux-build/procps/src/w.c:584:9: error: use of undeclared identifier '__UT_HOSTSIZE'
                             UT_HOSTSIZE);
                             ^
/home/builder/.termux-build/procps/src/w.c:76:22: note: expanded from macro 'UT_HOSTSIZE'
#       define UT_HOSTSIZE __UT_HOSTSIZE
                           ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make[2]: *** [Makefile:1558: w.o] Error 1
make[2]: *** Waiting for unfinished jobs....
1 warning generated.
1 warning generated.
make[2]: Leaving directory '/home/builder/.termux-build/procps/build'
make[1]: *** [Makefile:1894: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/procps/build'
make: *** [Makefile:1062: all] Error 2
```

### What steps will reproduce the bug?

./build-package.sh bash

### What is the expected behavior?

_No response_

### System information

termux-info:

```
no
```

--- a/packages/procps/w.c.patch
+++ b/packages/procps/w.c.patch
@@ -1,5 +1,15 @@
---- ./w.c.orig	2021-02-09 10:11:25.000000000 +0000
-+++ ./w.c	2021-05-07 05:42:59.852135096 +0000
+--- a/w.c
++++ b/w.c
+@@ -57,6 +57,9 @@
+ #include <unistd.h>
+ #ifdef HAVE_UTMPX_H
+ #	include <utmpx.h>
++#ifdef __ANDROID__
++#	include <utmp.h>
++#endif
+ #else
+ #	include <utmp.h>
+ #endif
 @@ -207,13 +207,6 @@
  
  	if (ip_addresses) { /* -i switch used */

